src/ui/SmartSnippet/SmartSnippet.ts

```diff
@@ -186,7 +194,7 @@ export class SmartSnippet extends Component {
   private buildHeightLimiter() {
     return (this.heightLimiter = new HeightLimiter(
       this.shadowContainer,
-      this.snippetContainer,
+      this.shadowContainer.childNodes.item(0) as HTMLElement,
```

Comment:
This is the actual fix. HeightLimiter was limiting to the size of `snippetContainer`, which is slightly smaller.